### PR TITLE
add presevation evoker spell ids to awcSpells

### PR DESCRIPTION
**Pull request by KMontag42**
current:
none of the big prevoker cooldowns are shown on the nameplates in the combat replay view

desired:
get prevoker spells into the combat replay view's nameplates

notes:
let me know if there is anything else I need to do.
it seems like I may need to re-generate the `spells.json` and related files via the `generateSpellsData` script, but I wasn't sure if that was a part of CI or something that PR contributors usually do.
happy to run the regeneration if needed :)

spells added:
```
    '363916', // obsidian scales
    '370553', // tip the scales
    '374348', // renewing blaze
    '363534', // rewind
    '357170', // time dilation
    '370960', // emerald communion
```

if there are any edits to this list that you'd like to see happy to change as well.

<3

### CODE REVIEW

**Review by kunchenguid — APPROVED** [review]

**Comment by KMontag42** [comment]
example screenshot

prevoker CDs are there and handled properly. woo!

File: packages/shared/src/data/spells.json [hunk 1]
```diff
@@ -165,9 +165,6 @@
   "11327": {
     "type": "buffs_defensive"
   },
-  "12042": {
```
Comment (kunchenguid) [inline packages/shared/src/data/spells.json:168]: were these manual changes or did you run the refresh script? i'm checking whether the removals are ok

File: packages/shared/src/data/spellEffects.json [hunk 2]
```diff
@@ -1273,5 +1273,47 @@
     "charges": {},
     "durationSeconds": 4,
     "cooldown": null
+  },
+  "357170": {
+    "spellId": "357170",
+    "name": "Time Dilation",
+    "charges": {},
+    "cooldownSeconds": 60
```
Comment (kunchenguid) [inline packages/shared/src/data/spellEffects.json:1281]: did you manually correct the field name? might be good to just fix the script

File: packages/shared/src/data/spells.json (hunk 1 above)
Comment (KMontag42) [reply packages/shared/src/data/spells.json:168]: from the refresh script

**Review by kunchenguid — APPROVED** [review]

**Pull request merged**